crashsimilarity/downloader.py

```diff
@@ -1,6 +1,6 @@
 import os
-from datetime import timedelta
 import logging
+from datetime import timedelta
```

Comment:
Could you move this back? I'd like to have clean history and so avoid unrelated changes

Reply:
Done